BannerlordTwitch/BannerlordTwitch/_Module/ModuleData/Languages/DE/strings-misc-DE.xml

```diff
@@ -1,601 +1,719 @@
+        channel point redemption &apos;Adopt a Hero&apos;. -->
+        <string id="rElL5v6I"
+            text="Bannerlord Twitch (BLT) ist ein Twitch-Integration für Mount &amp;amp; Blade II: Bannerlord.&lt;br&gt; Als Zuschauer kannst du Kanalpunkte-Belohnungen (sofern verfügbar) und Chat-Befehle verwenden, um mit dem Spiel zu interagieren, während der Streamer spielt.&lt;br&gt; Die Hauptfunktion von BLT besteht darin, dass man Helden im Spiel übernehmen kann, d. h. ein Charakter im Spiel wird deinem Twitch-Namen zugewiesen und steht für dich zur weiteren Interaktion zur Verfügung.&lt;br&gt; Einige Beispiele für Dinge, die du mit deinem Helden tun kannst, sind das „Beschwören“ des Helden in Schlachten, an denen der Streamer teilnimmt, die Teilnahme an Zuschauerturnieren, das Verbessern der Ausrüstung und die Auswahl der „Klasse“ deines Helden.&lt;br&gt; Wenn du dir nicht sicher bist, wo du anfangen sollst, beginne mit der Einlösung von Kanalpunkte-Belohnung „Erstelle einen Helden“."
+            original="Bannerlord Twitch (BLT) is a Twitch Integration mod for Mount &amp; Blade II: Bannerlord.&lt;br&gt;As a viewer you can use channel point rewards (if available), and chat commands to interact with the game while the streamer is playing.&lt;br&gt;The primary feature of BLT is allowing you to 'adopt' a hero in the game, i.e. a character in the game will be assigned to you, given your Twitch name, and be available for you to interact with further.&lt;br&gt;Some examples of things you can do with your hero include 'summoning' your hero into battles that the streamer is taking part in, joining viewer tournaments, upgrading equipment, selecting your heroes 'class'.&lt;br&gt;If you aren't sure where to start, begin by using the channel point redemption 'Adopt a Hero'." />
         <!-- Command -->
         <!-- Bot command definition -->
-        <string id="8SbGSWFY" text="Bot-Befehlsdefinition" original="Bot command definition"/>
+        <string id="8SbGSWFY" text="Bot-Befehlsdefinition" original="Bot command definition" />
         <!-- Name -->
-        <string id="D1lt1bSQ" text="Name" original="Name"/>
+        <string id="D1lt1bSQ" text="Name" original="Name" />
         <!-- The command itself, not including the ! -->
-        <string id="u1Ua0LXJ" text="Der Befehl selbst, ohne das !" original="The command itself, not including the !"/>
+        <string id="u1Ua0LXJ" text="Der Befehl selbst, ohne das ! (Ausrufezeichen)"
+            original="The command itself, not including the !" />
         <!-- HideHelp -->
-        <string id="TpnV5Mpi" text="AusblendenHilfe" original="HideHelp"/>
+        <string id="TpnV5Mpi" text="HilfeAusblenden" original="HideHelp" />
         <!-- Hides the command from the !help action -->
-        <string id="0AZVaBQN" text="Versteckt den Befehl aus der Aktion !Hilfe" original="Hides the command from the !help action"/>
+        <string id="0AZVaBQN" text="Versteckt den Befehl aus der Aktion !Hilfe"
```

Comment:
@billw2012 can `!Hilfe` be a variable? I noticed that the streamer, where I tested the changes, named this command `!help`